### PR TITLE
Add CLA

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

**Review by moisses89 — APPROVED** [review]

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: 'https://safe.global/cla/'
+          # branch should not be protected
+          branch: 'cla-signatures'
+          allowlist: hectorgomezv,moisses89,luarx,fmrsabino,luarx,rmeissner,Uxio0,*bot # may need to update this expression if we add new bots
```
Comment (fmrsabino) [inline .github/workflows/cla.yml:25]: I would remove the `*bot` for this one.

Comment (fmrsabino) [inline .github/workflows/cla.yml:25]: nitpick: `luarx` appears twice

**Review by fmrsabino — COMMENTED** [review]
Small comments.

Reminder to set `PERSONAL_ACCESS_TOKEN`

File: .github/workflows/cla.yml (hunk 1 above)
Comment (Uxio0) [reply .github/workflows/cla.yml]: Agree, done

Comment (Uxio0) [reply .github/workflows/cla.yml]: Fixed

**Comment by Uxio0** [comment]
> Reminder to set `PERSONAL_ACCESS_TOKEN`

Sure, it's set now

**Pull request merged**